pkg/monitor/cluster/clusterversions.go

```diff
@@ -34,13 +34,28 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 		}
 	}
 
+	// help determine which action should be taken after the RP is deployed
+	// possible scenarios:
+	//   - update the document to be on the same version as RP
+	//   - when document is OK update operator if did not picked automatically
+	rpAction := ""
+	switch {
+	case version.GitCommit != mon.oc.Properties.ProvisionedBy:
+		// resourceProviderVersion != provisionedByResourceProviderVersion
+		rpAction = "put/patch"
+	case mon.oc.Properties.ProvisionedBy != operatorVersion:
+		// provisionedByResourceProviderVersion != operatorVersions
+		rpAction = "deploy operator"
```

Comment:
As discussed I dont think this is needed. 
PutOrPatch deploys operators. And if those graphs do not match - we have an issue. I think we have all data in the current metrics for this.
If you think otherwise (we might be miscommunicated), let's discuss on the scrum. Otherwise - close?

Reply:
I have updated the code, to what we have discussed.